libs/oidc/common/src/lib/services/user/user.service.ts

```diff
@@ -182,15 +182,16 @@ export class UserService {
     const resetRequest = await this.passwordResetRepo.create(_resetRequest);
     resetRequest.setToken();
 
-    // TODO: Is this subscribe a potential source of problems later on? Should we have to unsubscribe?
-    this.httpService.get(`${this.IPSTACK_URL}${ip}?access_key=${this.IPSTACK_APIKEY}`).subscribe((observer) => {
-      const location = observer.data.country_name;
+    // TODO: Disabled for now because of too many API calls to ipstack; maybe we investigate other providers or scrap this functionality
+    // Is this subscribe a potential source of problems later on? Should we have to unsubscribe?
+    // this.httpService.get(`${this.IPSTACK_URL}${ip}?access_key=${this.IPSTACK_APIKEY}`).subscribe((observer) => {
+    //   const location = observer.data.country_name;
```

Comment:
It shouldn't be.

 `.get()` should be a unicast observable which means it should not keep emitting values past the initial time of execution.

Only way many API requests are going out is if the `sendPasswordResetEmail` method is called many times.

Reply:
I don't know where it's coming from but I do get emails from ipstack saying I'm using almost all of my free credits every month: "Please be aware that you have reached at least 101% of your available API Request volume. ". Unless someone using my API key I don't know where that usage is coming from.